### ISSUE TITLE
Realistic sky-sun brightness at night

### ISSUE THREAD

**Issue by Macintosh-Fan** [X0nk/Bliss-Shader#360]
Example:
The sun is 8.5° below the horizon in the image below, and I think the sky should only be this bright when the sun is at like 1° below the horizon.
![8.5-deg](https://github.com/user-attachments/assets/0f99f95e-f12c-4f58-a98e-800a76fbd34c)

Another example:
There should be no daylight visible to regular human eyes when the sun is 17° below the horizon in this image; maybe this ought to be how bright the sky is around 8.5° below the horizon instead.
![17.5-deg](https://github.com/user-attachments/assets/35ac682b-3402-42d7-a876-6efe5c273c9c)

**Comment by X0nk** [comment]
yeah, i had some stuff going on with the atmosphere simulation to extend the "blue hour". however it kinda ruins sunsets, so im removing it. it has already been removed in the unstable development branch, so it should be coming to the main branch sometime soon

**Comment by Macintosh-Fan** [comment]
Not going to be fixed?

**Comment by X0nk** [comment]
to be clear, it was not broken, it was designed to be that way... it just wasnt good design. the status hasnt changed from what i said before. the changes from the unstable branch will eventually move to the stable branch.

the reason ive closed it is because i had 160 open issues, and honestly, i cannot be bothered to properly sort through and check them all properly, they range from 2 years ago to now, so i just thought fuck it, close them all and see whats comes back.

in my mass closing of issues i have closed some suggestions/discussion threads inappropriately which i will open again if they come back to life (as you have done right now by replying)

**Comment by joshthebossxl** [comment]
Hello, I also noticed this issue, and I actually brought it up in the discord, almost a year ago, I was looking through the files trying to find a way to edit it myself for example, but I have not found it yet! If this does prove too much of a hassle to get accurate to what we have in the world today, could there be options for someone to change it in the settings?
